atmat/lattice/atfitchrom.m

```diff
@@ -13,19 +15,16 @@
 %   Numeric array: list of selected elements in RING
 %   Cell array: All elements selected by each cell
 
```

Comment:
may also the optional inputs could be in the help

Reply:
Done